packages/inventory/package.json

```diff
@@ -26,8 +26,8 @@
     },
     "homepage": "https://github.com/RedHatInsights/frontend-components/tree/master/packages/inventory#readme",
     "peerDependencies": {
-        "react": "^16.14.0",
-        "react-dom": "^16.14.0",
+        "react": ">=16.5.1 || >=17.0.0",
+        "react-dom": ">=16.5.1 || >=17.0.0",
```

Comment:
I don't think we can go below 16.8.x, the package is using react hooks.

Reply:
Fixed.